.circleci/config.yml

```diff
@@ -0,0 +1,25 @@
+version: 2.1
+jobs:
+  build:
+    working_directory: ~/repo
```

Comment:
Why `~/repo`? I didn't see anything in the docs.

Reply:
That was the go sample config they provide while you configure a new build.

Reply:
I went through the docs too, and even I did not see `working_directory` being set anywhere. And it doesn't make sense to set it up at this point, so removing it.